CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ _None_
 * Images: new dot-syntax template, use `dot-syntax-swift3` or `dot-syntax` (for Swift 2.3).  
   [David Jennes](https://github.com/djbe)
   [#206](https://github.com/AliSoftware/SwiftGen/pull/206)
+* Stencil: added two new tags `define` and `call`, see the documentation for in depth explanations on how to use them. Reworked the "dot-syntax" and "structured" templates to use them, which removes the limitation of 5-level deep structures.  
+  [David Jennes](https://github.com/djbe)
+  [#232](https://github.com/AliSoftware/SwiftGen/pull/232)
```

Comment:
Wondering if it wouldn't be more readable to split that CHANGELOG entry in two, one in "Internal Changes" about the new tags define & call (well that's not totally internal, as it opens those tags for people doing custom templates, so maybe not such a good idea, but …), and another to tell that the 5-level limitation has been lifted?

Not really sure, just opening the debate.